src/View/StringTemplate.php

```diff
@@ -315,6 +315,9 @@ protected function _formatAttribute($key, $value, $escape = true)
         }
         $truthy = [1, '1', true, 'true', $key];
         $isMinimized = isset($this->_compactAttributes[$key]);
+        if(!preg_match_all('/\A(\w|[.-])+\z/', $key)) {
```

Comment:
Pretty sure this is supposed to be `if (` with a space.

Reply:
Do you need preg_match_all or would preg_match work?

Reply:
not sure, im no regex hero, if you think `preg_match` would do, i can change that